Configuration/TypoScript/setup.txt

```diff
@@ -216,6 +216,11 @@ plugin.tx_t3extblog {
 				# checks if client accepts cookies
 				cookie = 2
 			}
+
+            privacyPolicy {
+                enabled = {$plugin.tx_t3extblog.settings.privacyPolicy.enabled}
```

Comment:
We probably want to have this more specific to enable / disable specific checkboxes. So probably more like `enableForBlogSubscription`?